modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * virt//support/monitoring/virt-running-cluster-checkups.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-building-vm-containerdisk-image_{context}"]                                  
+= Building a container disk image for RHEL virtual machines
+
+You can build a custom RHEL 8 OS image in `qcow2` format and use it to build a container disk image for RHEL virtual machines (VMs). You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmContainerDiskImage` attribute of the DPDK checkup config map. 
+
+Run the following procedure in the image builder VM to build a container disk image for RHEL 8.7 VMs with 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory. 
```

Comment:
I suggest to define the term "image builder VM" as a RHEL 8 VM used to build custom RHEL images.